README.md

```diff
@@ -1,39 +1,40 @@
-# Konnectors
+![Travis build status shield](https://img.shields.io/travis/cozy-labs/konnectors.svg)
+![NPM release version shield](https://img.shields.io/npm/v/konnectorsh.svg)
+![Github Release version shield](https://img.shields.io/github/release/cozy-labs/konnectors.svg)
+![NPM Licence shield](https://img.shields.io/npm/l/konnectorsh.svg)
 
-[![Build Status](https://travis-ci.org/cozy-labs/konnectors.svg)](https://travis-ci.org/cozy-labs/konnectors)
 
-Collection of connectors to fetch data from different websites and save them
-into your Cozy.
+[Cozy][cozy] MyAccounts
+=======================
 
-If you want to build your own connector, here is [a complete documentation](https://github.com/cozy-labs/konnectors/wiki)
 
-# Contributing rules
+What's Cozy?
+------------
 
-Before opening any pull requests make sure that you follow these rules:
+![Cozy Logo](https://cdn.rawgit.com/cozy/cozy-guidelines/master/templates/cozy_logo_small.svg)
 
-* Make your pull requests on the master branch.
-* Follow the [Cozy Guidelines](https://github.com/cozy/cozy-guidelines).
-* Make sure that `npm run lint` doesn't return any errors.
-* Do not include the build in your pull request.
-* For konnector addition, make sure that the following locale keys are present:
- * 'konnector description my_connector_name': explains what the connector does
- * Notification texts: if your connector pops up notifications on completed fetch,
-   it requires a localized text (use any of the existing docTypes notification or create your own key).
-* You have updated the connector list (if relevant)
-* Explain in the Pull Request which problem is solved and how you fixed it (plateform changes).
+[Cozy] is a platform that brings all your web services in the same private space.  With it, your webapps and your devices can share data easily, providing you with a new experience. You can install Cozy on your own hardware where no one's tracking you.
 
-# Available connectors
+
+What's MyAccounts (previously known as Konnectors)?
+------------------
+
+Collection of connectors to fetch data from different websites and save them into your Cozy.
+
+
+Available connectors
+--------------------
 
 The connectors marked with :x: are known as currently broken.
 
-## Bills
+### Bills
```

Comment:
Kind of long-term reflexion but should not this section be partly automated. Typically by replacing `:x` by links to images _à la_ "build passed" by Travis? Would ease to keep track of current konnectors status, without having to commit each time one is broken, and could be plugged to a reporting service where users could flag konnectors as being invalid.